package.json

```diff
@@ -2,11 +2,13 @@
   "name": "safe-apps",
   "private": true,
   "scripts": {
+    "build": "yarn workspaces run build",
+    "lint:check": "eslint './apps/**/*.{js,jsx,ts,tsx}'",
+    "lint:fix": "yarn lint:check --fix",
```

Comment:
It would be nice to also workspace-ify the listing, so that we can the apps aren't affecting each other.

Reply:
What do you mean by affecting each other? It doesn't make sense to use different style guides in a monorepo, does it? I hope I got it right

Reply:
I meant when you run the fix/prettier, it would be nice to not fix/prettify the apps you haven't touched. But yarn workspaces probably won't help here anyway.
Please disregard my comment.